pkg/templates/privilegedports/template.go

```diff
@@ -0,0 +1,40 @@
+package privilegedports
+
+import (
+	"fmt"
+
+	"golang.stackrox.io/kube-linter/pkg/check"
+	"golang.stackrox.io/kube-linter/pkg/config"
+	"golang.stackrox.io/kube-linter/pkg/diagnostic"
+	"golang.stackrox.io/kube-linter/pkg/objectkinds"
+	"golang.stackrox.io/kube-linter/pkg/templates"
+	"golang.stackrox.io/kube-linter/pkg/templates/privilegedports/internal/params"
+	"golang.stackrox.io/kube-linter/pkg/templates/util"
+	v1 "k8s.io/api/core/v1"
+)
+
+func init() {
+	templates.Register(check.Template{
+		HumanName:   "Privileged Ports",
+		Key:         "privileged-ports",
+		Description: "Flag privileged ports",
+		SupportedObjectKinds: config.ObjectKindsDesc{
+			ObjectKinds: []string{objectkinds.DeploymentLike},
+		},
+		Parameters:             params.ParamDescs,
+		ParseAndValidateParams: params.ParseAndValidate,
+		Instantiate: params.WrapInstantiateFunc(func(_ params.Params) (check.Func, error) {
+			return util.PerContainerCheck(func(container *v1.Container) []diagnostic.Diagnostic {
+				var results []diagnostic.Diagnostic
+				for _, port := range container.Ports {
+					if int(port.ContainerPort) >= 0 && int(port.ContainerPort) <= 1024 {
```

Comment:
I think these should be exclusive?
```suggestion
					if int(port.ContainerPort) > 0 && int(port.ContainerPort) < 1024 {
```

Reply:
Not that it matters, a port of 0 is a mistake anyway.